src/Breadcrumb.js

```diff
@@ -1,27 +1,43 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import BreadcrumbItem from './BreadcrumbItem';
 import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
 
+const propTypes = {
+  /**
+   * ARIA label for the nav, if no label is provided
+   * it will default to "breadcrumbs".
```

Comment:
no need to list the default the docs generation will pick up the defaultProps for it.